wordpress-mobile/WordPress-Android
# Blogging prompts create local notification trigger introduction dialog

Fixes #16263

Adds the blogging prompts onboarding local notification. Notification dismiss button will be handled in a [separate issue](https://github.com/wordpress-mobile/WordPress-Android/issues/16295).

To test:
- Enable the blogging prompts notification feature flag (see `BloggingPromptsNotificationConfig`)
- Open app logged in and with a site selected
- Click on the blogging prompts onboarding notification OR in the "Learn more" button
- Blogging prompts onboarding dialog should be seen

<p float="left">
<img width="313" alt="Screen Shot 2022-04-10 at 9 56 53 PM" src="https://user-images.githubusercontent.com/14964993/162652371-1e2a112a-78be-47f2-8b69-49a61b22c7af.png">
<img width="314" alt="Screen Shot 2022-04-10 at 10 50 10 PM" src="https://user-images.githubusercontent.com/14964993/162652375-67b953b2-1e9b-415a-b358-65033bc1b40a.png">
</p>

## Regression Notes
1. Potential unintended areas of impact
Scheduled notifications in `WPMainActivity`

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Running `WPMainActivity` with different values for the new feature flag

3. What automated tests I added (or what prevented me from doing so)
`TODO`

PR submission checklist:
